src/app/checklists/items-list/item/item.component.html

```diff
@@ -74,6 +74,12 @@
     </button>
   </div>
   <div class="item" [class.item-centered]="it.centered" tabindex="0">
+    <span class="prompt-spacer" [style.padding-left]="it.centered ? '' : '10px'"></span>
+    @if (!it.centered) {
+      @for (_ of [].constructor(it.indent); track $index) {
+        <span class="prompt-tabulator">⟷</span>
```

Comment:
Thought on how to make this even less visually intrusive, keeping the default visualization close to the avionics: what if it showed up only on hover? (you don't even need code for it, CSS `:hover:before` can apply `content` like it does for the prefixes and checkboxes)